api/v1beta2/moduletemplate_types.go

```diff
@@ -126,6 +126,30 @@ type ModuleTemplateSpec struct {
 
 	// CustomStateCheck is deprecated.
 	CustomStateCheck []*CustomStateCheck `json:"customStateCheck,omitempty"`
+
+	// Info is the information about the module.
+	// +optional
+	Info ModuleInfo `json:"info,omitempty"`
```

Comment:
```suggestion
	// Info contains metadata about the module.
	// +optional
	Info ModuleInfo `json:"info,omitempty"`
```

Okay as is, but I like the way you phrased it in the docs.